seat_service/src/bin/seat_service/seat_data_feeder.cc

```diff
@@ -54,10 +54,10 @@ SeatDataFeeder::SeatDataFeeder(std::shared_ptr<SeatAdjuster> seat_adjuster, std:
             broker_feeder::createDatapoint(1U),
             "Number of seats in row 1"},
         {seat_pos_name,
-            DataType::UINT8, // Changed from UINT32 to match VSS 3.0
+            DataType::UINT16, // Changed from UINT32 to match VSS 3.0
             ChangeType::ON_CHANGE,
             broker_feeder::createNotAvailableValue(),
-            "Longitudinal position of overall seat"}
+            "An arbitrary name - not necessarily same as in VSS"}
```

Comment:
What is the reason to use custom description here?

Reply:
But not necessarily to change it, so I reverted this change

Reply:
All feeders/providers register their datapoints at the moment.
Maybe they can't change already registered metadata, but databroker does not fail.
It's more confusing to add such VSS centric description to the service. I'm also fine to use an empty description.

Reply:
As far as I know there will be an error if the signal does not already exist. anyhow, for now this change has been removed from the PR, so it remains as "Longitudinal position of overall seat"